index.html

```diff
@@ -13,7 +13,123 @@
   <![endif]-->
 </head>
 
-<body>
-  <h1>My Resume</h1>
+<body></div>
+  <header>
+
+      <div class="container">
+
+          <div class="navigation">
+            <nav>
+              <a href="#">Home</a>
+              <a href="#">About</a> 
+              <a href="#">Portfolio</a>
+              <a href="#">Contact</a>
+            </nav>
+</header>
+
+<div class="cta">
+    <section class="cta-text">
+      <h1>Justin<br>
+      Canady<br>
+      </h1>
+      <button type="work-together">Lets Work Together</button>
+    </section>
+    <!-- Image -->
+    <img class="bio" src="img/coding.jpg" alt="Web Dev">   
+  </div>
+
+<div class="main-body">
+<!-- Top Text -->
+
+<section class="top-content">
+  <div class="text-container">
+    <h2>Intro</h2>
+    <p>Building state-of-the-art, easy to use, user-friendly websites and applications is truly a passion of mine and I am confident I would be an excellent addition to your organization. In addition to my knowledge base, I actively seek out new technologies and stay up-to-date on industry trends and advancements. This has allowed me to stay ahead of the curve and deliver exceptional work to all of my employers, including those I've worked for on a project basis. I’ve attached a copy of my resume detailing my experience, along with links to websites and applications I’ve had the honor of working on.</p>
+  </div>
+
+</section>
+
+<section class="skills">
+  <div class="skill-hist">
+<h2>Key Skills</h2>
+
+<p>Respected Full-Stack Web Developer with 14-plus years of experience in collaborating with creative and development teams on the execution of ideas and projects.</p>
+
+<p>
```

Comment:
this paragraph tag is unnecessary.  The list tags are enough to wrap this information.